draft-ietf-webtrans-http3.md

```diff
@@ -76,7 +76,55 @@ server.  An HTTP/3 server is the server that terminates HTTP/3 connections; a
 WebTransport server is an application that accepts WebTransport sessions, which
 can be accessed via an HTTP/3 server.
 
-# Protocol Overview
+# Overview
+
+## WebTransport, QUIC and HTTP/3
+
+"QUIC: A UDP-Based Multiplexed and Secure Transport" {{!RFC9000}}
+defines QUIC security, stream multiplexing, reliable and ordered streams,
+stream HoL blocking avoidance, flow control, and congestion control.
+The transport layer provides these services to applications but does
+not constrain how streams are used.
+
+HTTP is an application-layer protocol, defined by "HTTP Semantics" {{!RFC9110}}.
+HTTP/3 has specific features that are not part of the HTTP semantics: QPACK
```

Comment:
I think that you want to say that "HTTP/3 is an adaptation of HTTP so that it can use QUIC, where previous HTTP versions used TCP."

I would not talk so much about header compression or the other stuff.

I would also move the second sentence from the next paragraph up to this one.